x-pack/plugins/task_manager/server/task_running/task_runner.ts

```diff
@@ -162,6 +165,7 @@ export class TaskManagerRunner implements TaskRunner {
   private eventLoopDelayConfig: EventLoopDelayConfig;
   private readonly taskValidator: TaskValidator;
   private readonly claimStrategy: string;
+  private currentPollInterval?: number;
```

Comment:
should we set a default in case something goes wrong with the observable?

Reply:
Makes sense, I defaulted it to the config.poll_interval in this commit: https://github.com/elastic/kibana/pull/190093/commits/d30bbfc5c6c0d97bc1ba1ea1e51ef34913f7ccf9